_data-prepper/pipelines/configuration/processors/delete-entries.md

```diff
@@ -8,7 +8,11 @@ nav_order: 110
  
 # Delete entries processor
 
-The `delete_entries` processor deletes entries, such as key-value pairs, from an event. You can define the keys you want to delete in the `with-keys` field following `delete_entries` in the YAML configuration file. Those keys and their values are deleted. 
+The `delete_entries` processor removes entries, such as key-value pairs, from an event. Use the `with_keys` field to specify
+the exact keys to delete. To delete keys that match a regular expression pattern, use the `with_keys_regex` field. You can
+prevent deletion of specific events when using regular expressions by configuring the `exclude_from_regex` field.
+
+To use both `with_keys` and `with_keys_regex` in the same configuration, define them under the `entries` field.
```

Comment:
I think this is a little confusing. Do you mean to say that the pipeline author would need to create two different `entries`?